OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -466,7 +475,11 @@ public override bool Tick(Actor self)
 					mobile.SetTerrainRampOrientation(orientation);
 				}
 
-				mobile.Facing = WAngle.Lerp(FromFacing, ToFacing, progress, Distance);
+				if (TurnsWhileMoving)
```

Comment:
```suggestion
				mobile.Facing = TurnsWhileMoving
					? Util.TickFacing(mobile.Facing, FromToYaw + Move.ActorFacingModifier, mobile.TurnSpeed)
					: WAngle.Lerp(FromFacing, ToFacing, progress, Distance);
```

Use this structure. Same thing above